best-practices.js

```diff
@@ -98,4 +98,63 @@ module.exports = {
       },
     ],
   },
+  overrides: [
+    {
+      files: ['**/*.ts?(x)'],
+      rules: {
+        'no-undef': 'off', // ts(2304)
+
+        'default-param-last': 'off',
+        '@typescript-eslint/default-param-last': 'off',
+
+        'dot-notation': 'off',
+        '@typescript-eslint/dot-notation': 'error',
+
+        'no-empty-function': 'off',
+        '@typescript-eslint/no-empty-function': 'off',
+
+        'no-implied-eval': 'error',
+        '@typescript-eslint/no-implied-eval': 'error',
+
+        'no-invalid-this': 'off',
+        '@typescript-eslint/no-invalid-this': 'error',
+
+        'no-loop-func': 'off',
+        '@typescript-eslint/no-loop-func': 'error',
+
+        'no-magic-numbers': 'off',
+        '@typescript-eslint/no-magic-numbers': 'off',
+
+        'no-redeclare': 'off',
+        '@typescript-eslint/no-redeclare': 'error',
+        // '@typescript-eslint/no-redeclare': 'off', // ts(2451)
```

Comment:
Don't know what to do with this one as it would be handled by TS itself according to [`plugin:@typescript-eslint/recommended`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/eslint-recommended.ts)

Reply:
I don't want to just enable their recommended rules. I want to configure each rule myself. I think that's what we have now.

Reply:
These rules aren't in the [Supported Rules](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/README.md#supported-rules) but in the [Extension Rules](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/README.md#extension-rules) (which is the TypeScript variant of the ESLint rule) though